### PR TITLE
Write a blog post for week 48, 2020

### PR DESCRIPTION
**Pull request by csomh**
Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

File: content/posts/November-2020.md [hunk 1]
```diff
@@ -23,16 +23,36 @@ draft: false
...
   our new users ([packit#1024](https://github.com/packit/packit/pull/1024)).
 
+## Week 48 (November 23rd - November 27th)
+
+- [Laura] [fixed a bug], where Packit Service failed to trigger Copr builds
+  before triggering the tests in case triggering Copr builds was configured
+  for a different event.
+- Thanks to [Matej], Packit Service will now [comment on commits] in case jobs
+  failed after a merge. This way failures for jobs triggered by branch events
+  (merge or push) should become more noticeable.
```
Comment (csomh) [inline content/posts/November-2020.md:44]: @mfocko Is this correct?

Comment (mfocko) [reply content/posts/November-2020.md:44]: Not really, it comments when there are unsupported targets, failed update of copr project or no builder permissions

Comment (mfocko) [reply content/posts/November-2020.md:44]: tbh it's a nice idea though, shouldn't be that hard to add :+1:

**Review by TomasTomecek — APPROVED** [review]

File: content/posts/November-2020.md [hunk 2]
```diff
@@ -23,16 +23,35 @@ draft: false
...
   next to the test results. We hope this to be a more friendly behaviour for
   our new users ([packit#1024](https://github.com/packit/packit/pull/1024)).
 
+## Week 48 (November 23rd - November 27th)
+
+- [Laura] [fixed a bug], where Packit Service failed to trigger Copr builds
+  before triggering the tests in case triggering Copr builds was configured
+  for a different event.
+- Thanks to [Matej], Packit Service will now [comment on commits] in case Copr
+  builds fail after a merge, in order to indicate the reason for the failure.
```
Comment (csomh) [inline content/posts/November-2020.md:43]: @mfocko did I put it the right way? :slightly_smiling_face:

Comment (mfocko) [reply content/posts/November-2020.md:43]: Yes :+1:

**Review by mfocko — APPROVED** [review]

**Pull request merged**